### DISCUSSION TITLE
When using Vim in Tabby, encountered two issues.

### DISCUSSION THREAD

**Discussion by HaHa131221** [Eugeny/tabby#8497] [Q&A]
1. When using Vim in Tabby, the cursor has a problem when backspace is used.
![b1](https://github.com/Eugeny/tabby/assets/35452190/cca35282-a51a-479e-85aa-4afaab9b87e6)
![b2](https://github.com/Eugeny/tabby/assets/35452190/a0968918-6590-4b5d-8b5b-78193bf3fe25)

2. Enter content on a new line, and the following pattern prompts for issues.
![a1](https://github.com/Eugeny/tabby/assets/35452190/203c0e76-9c9c-42b7-b0c7-7a5d336ef3e7)
![a2](https://github.com/Eugeny/tabby/assets/35452190/0288e767-a40c-48bf-9704-15084d99b1a1)

**Comment by MrRohwei** [comment]
I have also encountered this problem. Has it been solved so far?

**Reply by HaHa131221** [reply]
Not yet

**Reply by MrRohwei** [reply]
I don't know when this problem can be solved.Are you Chinese?

**Reply by leonlau** [reply]
Tabby version: 1.0.221
target host LANG set to zh_CN.utf8
``` bash
LANG=zh_CN.utf8  vim test.txt
```
If you enter more than 4 characters, deleting with the backspace key will cause an issue.
